ui/src/components/home/Home.tsx

```diff
@@ -36,7 +36,7 @@ export default class Home extends Component<{}, HomeState> {
                 <div>
                     <Button variant="contained" size="large" color="primary" style={homeStyles.payButton} onClick={() => this.toggleDialog(true)}>Pay</Button>
                 </div>
-                <PaymentDialog isModalOpen={this.state.isPaymentModalOpen} toggleModalOpen={this.toggleDialog}/>
+                {this.state.isPaymentModalOpen ? <PaymentDialog isModalOpen={this.state.isPaymentModalOpen} toggleModalOpen={this.toggleDialog}/> : null}
```

Comment:
Not critical, just note.

If you have common functionality in several stories, it's better to implement base logic in the corresponding story and other stories should depend on it (I mean `branch` depending on the `Git`). Because it's a little bit confusing when I see the same code in the two branches, it would be better if one branch depends on another one. In your case: `SS-37` -> `SS-28`.

Reply:
Yep, we should first merge previous PR, then it'll be only 1 branch, but you can actually select changes in 1 commit
So your suggestion is to name branch like that:
SS-37 -> SS-28 ?

Reply:
No-no, I mean, we can change branch to which this one will be merged: `SS-28`.